website/docs/getting-started/for-designers.md

```diff
@@ -19,26 +18,27 @@ For documentation around broader Figma best practices within HashiCorp design te
 
 1. Open the Library modal via one of the following methods:
 
-    - Click the `Assets` panel in the left sidebar, then click the `book` icon to open the modal.
+   - Click the `Assets` panel in the left sidebar, then click the `book` icon to open the modal.
```

Comment:
[todo] with this change you have lost the list indentation (it should be 4 spaces to be a second-level item). Compare [the old page](https://helios.hashicorp.design/getting-started/for-designers#enabling-helios-libraries) with [the new one](https://hds-website-git-hds-4111-update-figma-links-hashicorp.vercel.app/getting-started/for-designers#enabling-helios-libraries).

Reply:
Fixed this formatting issue